src/seedu/addressbook/data/person/Tagging.java

```diff
@@ -0,0 +1,32 @@
+package seedu.addressbook.data;
+
+import java.util.ArrayList;
+
+public class Tagging {
+    
+    private static ArrayList<String> listOfTaggings;
```

Comment:
recollect the discussion during tutorial on why this is not the right place to maintain the tagging list.
please check and differentiate the 2 diagrams in the activity description for better understanding